.travis.yml

```diff
@@ -119,30 +119,38 @@ matrix:
         - ./.travis/install-cython-examples.sh
       script:
         - export PATH="$HOME/miniconda/bin:$PATH"
+        # The following is needed so cloudpickle can find some of the
+        # class definitions: The main module of tests that are run
+        # with pytest have the same name as the test file -- and this
+        # module is only found if the test directory is in the PYTHONPATH.
+        - export PYTHONPATH="$PYTHONPATH:./test/"
 
-        - python python/ray/common/test/test.py
-        - python python/ray/common/redis_module/runtest.py
-        - python python/ray/plasma/test/test.py
-        # - python python/ray/local_scheduler/test/test.py
-        # - python python/ray/global_scheduler/test/test.py
+        - python -m pytest python/ray/common/test/test.py
+        - python -m pytest python/ray/common/redis_module/runtest.py
+        - python -m pytest python/ray/plasma/test/test.py
+        # - python -m pytest python/ray/local_scheduler/test/test.py
+        # - python -m pytest python/ray/global_scheduler/test/test.py
 
         - python -m pytest python/ray/test/test_queue.py
         - python -m pytest test/xray_test.py
 
-        - python test/runtest.py
-        - python test/array_test.py
-        - python test/actor_test.py
-        - python test/autoscaler_test.py
-        - python test/tensorflow_test.py
-        - python test/failure_test.py
-        - python test/microbenchmarks.py
+        # The --assert=plain here is because pytest's assertion
+        # rewriting mechanism seems to mess up on this file,
+        # see https://api.travis-ci.org/v3/job/408275838/log.txt
```

Comment:
Don't link to this file because Travis only stores it for a short amount of time. Is there an issue perhaps?